gamemodes/terrortown/gamemode/shared/sh_main.lua

```diff
@@ -55,27 +55,20 @@ function GM:PlayerFootstep(ply, pos, foot, sound, volume, rf)
 end
 
 ---
--- Predicted move speed changes
+-- The Move hook is called for you to manipulate the player's MoveData.
+-- You shouldn't adjust the player's position in any way in the move hook. This is because due
```

Comment:
```suggestion
-- You shouldn't adjust the players position in any way in the move hook. This is due
```

idiot's apostrophe :P

Reply:
I have copied this exact text from the gmod wiki. However in english this is correct afaik.

Reply:
Maybe I should return my English certificates...